code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -126,6 +126,23 @@
 	body_parts_covered = CHEST|ARMS|GROIN|LEGS|FEET|HANDS
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 
+/obj/item/clothing/suit/toggle/owlwings
+	name = "owl cloak"
+	desc = "A soft brown cloak made of synthetic feathers. Soft to the touch, stylish, and a 2 meter wing span that will drive the ladies mad."
+	icon_state = "owl_wings"
+	item_state = "owl_wings"
+	togglename = "wings"
+	body_parts_covered = ARMS
+	armor = list(melee = 5, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)

```

Comment:
Let's not give powergamers a reason to get this suit every single time.

I know 5% melee resist isn't much, but it's still 5% melee resist; it's the difference in double eswords taking 3 hits to crit and 4.